src/style/placement.js

```diff
@@ -1,25 +1,26 @@
 // @flow
```

Comment:
Can we rename this file to `pauseable_placement.js` so the name doesn't collide with `src/symbol/placement.js`?